### ISSUE TITLE
Custom download path not working on Android tv 

### ISSUE THREAD

**Issue by dondeepak** [recloudstream/cloudstream#458]
### Steps to reproduce

Go to download path  chose custom and apply 

### Expected behavior

When choose custom and apply file manager will open to select the path

### Actual behavior

download path when i chose custom and apply nothing happens. 

### Cloudstream version and commit hash

4.0.1 ded96dd9

### Android version

Android 11

### Other details

 On Mi Android tv-  download path when i chose custom and apply nothing happens. 
But in Android mobile its working fine. 
Android version on tv  Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

**Comment by Blatzar** [comment]
You need an app to select the path, many android TVs lack a file picker compatible app. Most file managers cannot do it, but I'm pretty sure AnExplorer can handle it.

**Comment by dondeepak** [comment]
Yes i have AnExplorer its working for to restore data from backup but not for selecting custom path

**Comment by AustinAggiss** [comment]
Can confirm I also have this issue and cannot select a download path with any file manager.

**Comment by Terkyz** [comment]
I have the same issue but I'm using a phone, not an Android TV.
